Makefile

```diff
@@ -18,7 +18,7 @@ DOCKER_TARGETS=docker_build docker_push docker_tag
 
 all: $(SUBDIRS)
 clean: $(SUBDIRS) docu_clean
-$(DOCKER_TARGETS): $(SUBDIRS)
+$(DOCKER_TARGETS): yq_check $(SUBDIRS)
```

Comment:
I wonder if we should add it also to `all` ... I guess lot of people might just do `make all`

Reply:
Good point, and done.